src/main/java/dan200/computercraft/core/lua/LuaJLuaMachine.java

```diff
@@ -509,6 +510,11 @@ else if( object instanceof String )
             String s = object.toString();
             return LuaValue.valueOf( s );
         }
+        else if( object instanceof byte[] )
+        {
+            byte[] b = (byte[]) object;
+            return LuaValue.valueOf( Arrays.copyOf( b, b.length ) );
```

Comment:
In the rare case that the returner decides to mutate the value. `LuaValue.valueOf` doesn't copy it, so it seemed safer to do it here.